components/atoms/TranslatedText.tsx

```diff
@@ -0,0 +1,12 @@
+import { Typography, TypographyProps  } from '@mui/material';
+import React from 'react';
+
+// TS error for components
+// https://github.com/mui-org/material-ui/issues/19512#issue-558587710
+
+type Props =  TypographyProps & { children: string, component?: any };
+export const TranslatedText = ({ children, ...rest } : Props) => {
```

Comment:
I think it's customary to do `...props` here?

```suggestion
export const TranslatedText = ({ children, ...props } : Props) => {
```